src/app/components/annotator/text-annotator/text-annotator.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, OnInit } from '@angular/core';
+import { AnnotatorService } from 'src/app/services/annotator/annotator.service';
+
+@Component({
+  selector: 'evt-text-annotator',
+  templateUrl: './text-annotator.component.html',
+  styleUrls: ['./text-annotator.component.scss'],
+})
+export class TextAnnotatorComponent implements OnInit {
+
+  public selectedText: string;
+  public showAdder = false;
+  public menuPosition = {menu: {x: 0, y: 0}};
+
+  constructor(
+    private annotator: AnnotatorService,
+  ) {}
+
+  ngOnInit(): void {
```

Comment:
Why do you need this lifecycle hook?
Maybe you can put the body of ngOnInit into the constructor